Update `ember-cli-babel` to version 7

boris-petrov opened the pull request:
This is the only plugin that I use that still uses `ember-cli-babel` 6 so I decided to update it. Not sure if this is enough or more changes are needed to the plugin?

cc @vitch

boris-petrov committed:
Update `ember-cli-babel` to version 7

vitch commented:
Honestly I haven't used this plugin in many years. If you are using it and it still works for you with this change I'm happy to merge and release...

vitch commented:
Although it looks like Travis is unhappy because:

> The command "eval yarn --frozen-lockfile " failed.

boris-petrov commented:
I was just trying to figure that out. Any ideas what that is and why it fails?

vitch commented:
Actually - I think the real error is:

> The engine "node" is incompatible with this module. Expected version "8.* || 10.* || >= 12.*"

I set my node and yarn versions to the same as TravisCI via [volta](https://volta.sh/) (`6.17.1` and `1.3.2` respectively) and got the same error.

Maybe you can update `node_js:` in the `travis.yml` too?

boris-petrov committed:
Update Node.js' version used in Travis

boris-petrov commented:
Done. But this means a new major version. And while we're at it, we could do other version updates I guess?

boris-petrov committed:
Move `broccoli` to `devDependencies`

boris-petrov committed:
Use a supported fork of `webfonts-generator`

boris-petrov committed:
Add a `yarn: true` directive for Travis

boris-petrov committed:
Add `useYarn: true` to `config/ember-try.js`

boris-petrov commented:
@vitch - this is ready for merging. I moved a few things around and updated the used `webfonts-generator` library. The Node requirement had to be updated to version 12 because of [ttf2woff2](https://github.com/nfroidure/ttf2woff2/blob/master/CHANGELOG.md#breaking-changes) - [here](https://travis-ci.com/github/vitch/ember-cli-webfont/jobs/509817868) is a failing Travis build for proof.

I tried out the branch, it seems to work great. So a new (major) release will be appreciated! :) Thanks!
